.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false
+language: java
+matrix:
+  fast_finish: true
+  include:
+  - name: 'Java 8'
+    jdk: oraclejdk8
+#  - name: 'Java 11'
+#    jdk: openjdk11
+
+before_script:
+# required to avoid test failures from `Picked up _JAVA_OPTIONS: ...` message
+- unset _JAVA_OPTIONS
+script:
+- (cd jib-core; ./gradlew --console=plain --fail-fast --info --stacktrace build)
```

Comment:
That requires Travis to have write access to the gcr.io areas, right?

Reply:
Nope, the `jib-core` integration tests push to a local registry.

Reply:
Hmm, is there a way we can just put that binary in for travis to use? Otherwise, we could disable `DockerCredentialHelperIntegrationTest#testRetrieveGCR` for travis through some env var or sys prop.

Reply:
So the `jib-core` integration tests do access `gcr.io`, or is it that `docker-credential-gcr` is an unnecessary requirement?

Reply:
The `docker-credential-gcr` requirement is only to verify `DockerCredentialHelper` works as intended.

Reply:
Oh, I see. I think it's not difficult to install or download the `docker-credential-gcr` binary. That sounds like the easiest fix to enable the integration test.